### PR TITLE
Handle PHPUnit --stderr option

### PR DESCRIPTION
**Pull request by pczerkas**
Fix for tests, which depend on PHPUnit's `--stderr` option in order to stop PHPUnit from sending headers. E.g. functional test is using session_id() in setUp() to manipulate session ID. 

Without --stderr option passed to phpunit such test is failing with "headers already sent" due to PHPUnit's outputing on STDOUT.

### CODE REVIEW

**Comment by Jean85** [comment]
Also, you should merge or rebase master to fix the build.

**Comment by pczerkas** [comment]
Hi Jean,
I've made the requested changes. Please review them:)

For the changelog entry please add it, because I don't know in what version the changes would be relesed.

File: tests/Functional/Command/ParallelCommandTest.php [hunk 1]
```diff
@@ -81,9 +82,12 @@ public function testExecution(): void
         $this->assertContains(RaisingNoticeTestStub::class, $output);
         $this->assertContains(MissingProviderTestStub::class, $output);
         $this->assertContains(MySQLDeadLockTestStub::class, $output);
+        if (strpos((string) phpversion(), '7.1') !== 0) {
+            $this->assertContains(SessionTestStub::class, $output);
+        }
```
Comment (Jean85) [inline tests/Functional/Command/ParallelCommandTest.php]: Why is this needed?

Comment (pczerkas) [reply tests/Functional/Command/ParallelCommandTest.php]: PHP 7.1 behaved differently in CLI mode with regard to session_id(), headers_sent() etc.

I'm investingating it further - maybe this will resolve to changes in test class setUp() (and no PHP version checks elsewhere).

Comment (pczerkas) [reply tests/Functional/Command/ParallelCommandTest.php]: Alessandro, after investigation I have no other ideas to differentiate PHP versions behaviour than proposed in this PR. Please feel free to modify it;)

**Comment by Jean85** [comment]
Thank you for your contribution! I left the PHP 7.1 issue alone, I'll be dropping it in #149 soon.
